invenio_opendata/testsuite/data/cms/cms-learning-resources.xml

```diff
@@ -228,4 +228,47 @@
       <subfield code="b">Education</subfield>
     </datafield>
   </record>
+  <record>
+    <controlfield tag="001">61</controlfield>
+    <datafield tag="041" ind1=" " ind2=" ">
+      <subfield code="a">English</subfield>
+    </datafield>
+    <datafield tag="100" ind1=" " ind2=" ">
+      <subfield code="a">Lehti, Sami</subfield>
+      <subfield code="u">University of Helsinki</subfield>
+    </datafield>
+    <datafield tag="245" ind1=" " ind2=" ">
+      <subfield code="a">Computing Methods in High Energy Physics</subfield>
+    </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">University of Helsinki</subfield>
+      <subfield code="c">2014</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">This introductory course by S. Lehti is aimed for students with no prior knowledge of computing methods in high-energy physics. Prior knowledge of some programming language is needed. The course goes through basics of programming languages in high energy physics (FORTRAN, C++, python), software for analysis and visualisation (ROOT), some software for calculating cross sections and branching ratios, simulation of collisions (event generation), basics of CMS software (CMSSW) and an introduction to grid computing. This course is developed in the University of Helsinki for students interested in experimental particle physics.</subfield>
+    </datafield>
+    <datafield tag="516" ind1=" " ind2=" ">
+      <subfield code="a">All exercises can be done with the CMS open data image available from the portal.</subfield>
+    </datafield>
+    <datafield tag="556" ind1=" " ind2=" ">
+      <subfield code="a">Includes relevant warnings for the Univ. Helsinki specific part of the instructions (e.g. use of local computing resources).</subfield>
+    </datafield>
+    <datafield tag="653" ind1="1" ind2=" ">
+      <subfield code="a">External resource</subfield>
+    </datafield>
+    <datafield tag="693" ind1=" " ind2=" ">
+      <subfield code="a">CERN-LHC</subfield>
+      <subfield code="e">CMS</subfield>
+    </datafield>
+    <datafield tag="856" ind1="4" ind2=" ">
+      <subfield code="u">http://www.helsinki.fi/~slehti/ComputingMethodsInHEP/ComputingMethodsInHEP.html</subfield>
+      <subfield code="y">Go to the Computing Methods in High Energy Physics course</subfield>
+    </datafield>
+    <datafield tag="980" ind1=" " ind2=" ">
+      <subfield code="a">CMS-Learning-Resources</subfield>
+    </datafield>
+    <datafield tag="980" ind1=" " ind2=" ">
+      <subfield code="b">Education</subfield>
+    </datafield>
```

Comment:
Should this be research? This course is certainly useful for anyone for plan to start research. It is education for research....